benchmarks/operator_benchmark/pt/qtensor_method_test.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import operator_benchmark as op_bench
+import torch
+
+# Configs for pointwise and reduction unary ops
+qmethods_configs_short = op_bench.config_list(
+    attr_names=['M', 'N'],
+    attrs=[
+        [512, 512],
+    ],
+    cross_product_configs={
+        'dtype': [torch.quint8],
+        'contig': [True],
```

Comment:
Please add todo for non-contig
Also, might be a good idea to test this for int32 in addition